src/index.js

```diff
@@ -0,0 +1,146 @@
+const skyEmoji = document.querySelector('select');
+const reset = document.getElementById('reset');
+const currentWeather = document.getElementById('current-weather');
+const app = document.getElementById('weather-app');
+
+//dictionary for weather emojis
+const weatherImages = {
+    "Snow": [`🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨`, 'url(https://images.unsplash.com/photo-1576678052826-04fd6590483f?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTZ8fHNub3d5fGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=800&q=60)'],
+    "Mist": [`🌫☁️🌧☂☁️😶‍🌫️🌫️☁️🌧️☂`, 'url(https://images.unsplash.com/photo-1541675154750-0444c7d51e8e?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=860&q=80)'],
+    "Rain": [`🌧🌈⛈☔️🌧💧⛈🌧🌦☔️💧🌧🌧`, 'url(https://images.unsplash.com/photo-1503429134808-fdf0cd4e1bfa?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mzl8fHJhaW58ZW58MHx8MHx8&auto=format&fit=crop&w=800&q=60)'],
+    "Haze": [`😶‍🌫️🌁🌫😶‍🌫️🏭😷☁️🌁`, 'url(https://images.unsplash.com/photo-1571148433633-f62d3cdb5eee?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTJ8fHBvbGx1dGlvbnxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=800&q=60)'],
+    "Clouds": [`☀️ 🌤️ ⛅️ 🌥️ 🌤️ ⛅️ 🌥️ ☁️`, 'url(https://images.unsplash.com/photo-1524555259-3e4f9092f97b?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTJ8fGNsb3VkeXxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=800&q=60)'],
+    "Sunny": [`☀️☀️☀️☀️☀️☀️☀️☀️☀️`, 'url(https://images.unsplash.com/photo-1462524500090-89443873e2b4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80'],
+    "Clear": [`☀️☀️☀️☀️☀️☀️☀️☀️☀️`, 'url(https://images.unsplash.com/photo-1462524500090-89443873e2b4?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80'],
+    "Smoke": [`🚬😶‍🌫️🔥 🌫😶‍🌫️🏭😷🫁`, 'url(https://images.unsplash.com/photo-1571148433633-f62d3cdb5eee?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTJ8fHBvbGx1dGlvbnxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=800&q=60)'],
+    "Fog": [`😶‍🌫️🌁🌫😶‍🌫️🏭😷☁️🌁`, 'url(https://images.unsplash.com/photo-1485236715568-ddc5ee6ca227?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8Zm9nfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=800&q=60)'],
+    "Other": [`🌈☀️ 🌤️ ⛅️ 🌥️ 🌤️ ⛅️ 🌥️ ☁️`, 'url(https://images.unsplash.com/photo-1470240731273-7821a6eeb6bd?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8c3ByaW5nfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=800&q=60)']
+};
+
+
+const getCoordinates = async function () {
+    const coordinates = await axios.get(`http://localhost:5000/location?q=${search.value}`);
+    const lat = coordinates.data[0].lat;
+    const lon = coordinates.data[0].lon;
+    const name = coordinates.data[0].display_name;
+    console.log(coordinates);
+    return [lat, lon, name];
+};
+
+const getWeather = async function () {
+    const coordinates = await getCoordinates();
+    const weather = await axios.get(`http://localhost:5000/weather?lat=${coordinates[0]}&lon=${coordinates[1]}`);
+    return weather;
+};
```

Comment:
Really nice use of the `async & await` functionality in your `getWeather` and `getCoordinates` functions! It makes the code so much more concise and easier to read than promise chaining! ✨